src/test/java/roomescape/auth/jwt/JwtTokenExtractorTest.java

```diff
@@ -0,0 +1,84 @@
+package roomescape.auth.jwt;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.security.Keys;
+import java.security.Key;
+import java.util.Base64;
+import java.util.Date;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import roomescape.exception.custom.AuthorizationException;
+
+class JwtTokenExtractorTest {
+
+    private JwtTokenExtractor jwtTokenExtractor;
+    private Key key;
+
+    @BeforeEach
+    void setUp() {
+        key = Keys.secretKeyFor(SignatureAlgorithm.HS256);
+        String base64SecretKey = Base64.getEncoder().encodeToString(key.getEncoded());
+        jwtTokenExtractor = new JwtTokenExtractor(base64SecretKey);
+    }
```

Comment:
```suggestion
    private final Key key = Keys.secretKeyFor(SignatureAlgorithm.HS256);
    private final JwtTokenExtractor jwtTokenExtractor = new JwtTokenExtractor(
            Base64.getEncoder().encodeToString(key.getEncoded())
    );
```

junit5부터는 테스트의 라이프사이클이 메서드당이기 때문에 setUp을 할 필요가 없습니다.

https://junit.org/junit5/docs/current/user-guide/#writing-tests-test-instance-lifecycle

Reply:
setUp을 제거하였습니다!

알려주셔서 감사합니다 😊